kuksa-appmanager/README.md

```diff
@@ -1,11 +1,14 @@
 # Kuksa AppManager
 
-The AppManager is a python client that uses HawkBit artifacts to launch apps as Docker containers.
+The AppManager is a python client that uses HawkBit artifacts to launch apps as Docker containers or AGL widgets.
 
 Once started, the AppManager will check regularly with HawkBit for required actions, covering deployment and configuration data. Additionally a MQTT connection
 to Hono is used to receive `configuration change` events that trigger deployment actions.
 
-Currently, only the `docker-configuration.json` artifact is supported. The AppManager uses this JSON file to to pull images from a Docker registry and create containers.
+Currently, two types of app artifacts are supported. Docker containers and AGL widgets.
+A Docker container must contain a `docker-configuration.json` artifact.
```

Comment:
While you are at it... there is a long-standing typo here, actually the file for dockers needs to be called `docker-container.json`  (see https://github.com/eclipse/kuksa.invehicle/blob/0a3238abfc83950d2113baff82722aa49f637f37/kuksa-appmanager/kuksa/appmanager/hawkbit.py#L342)   Maybe you can change this in the REAMDE too, so I don't need a separate PR for that....

Reply:
No problem. Fixed